open-metadata-implementation/access-services/asset-lineage/asset-lineage-server/src/main/java/org/odpi/openmetadata/accessservices/assetlineage/handlers/AssetContextHandler.java

```diff
@@ -187,7 +186,7 @@ public RelationshipsContext buildAssetContext(String userId, EntityDetail entity
      */
     public Map<String, RelationshipsContext> buildColumnContext(String userId, LineageEntity lineageEntity)
             throws OCFCheckedExceptionBase {
-        if(!Arrays.asList(TABULAR_COLUMN, RELATIONAL_COLUMN).contains(lineageEntity.getTypeDefName())){
+        if (!Arrays.asList(TABULAR_COLUMN, RELATIONAL_COLUMN).contains(lineageEntity.getTypeDefName())) {
```

Comment:
@marius-patrascu we need to address this in the change you are working on with the TabulatFileColumn (we need to check if we can use repositoryHelper.isTypeOf to do the check with the super type). Please take this into account.